### ISSUE TITLE
Reduce Lambda Package size

### ISSUE THREAD

**Issue by DregondRahl** [maoosi/prisma-appsync#10]
After running prisma generate inside afterBundling(), we won't need the prisma package anymore. we could run yarn remove after that, also we won't need the prisma-appsync package if its being compiled. Removing them reduces the size significantly. 

Zipped file size reduced from 75MB to 19MB

```
              // added some other clean up also.
              "yarn prisma generate",
              "rm -rf node_modules/.prisma/client/query-engine-debian-openssl-1.1.x",
              "rm -rf node_modules/@prisma/engines-version/*",
              "rm -rf node_modules/@prisma/engines/*",
              "yarn remove prisma",
              "yarn remove prisma-appsync",

```

Is there any reason to keep the packages?

**Comment by maoosi** [comment]
No particular reason. In fact, I recently ran into an issue with bundle sizing myself and made some changes to the provided CDK template (https://github.com/maoosi/prisma-appsync/blob/next/boilerplate/cdk/index.ts#L110). I'll merge with the main branch soon.

After asking the question to Ryan Dsouza from the Prisma team, he told me we could safely do the following:

```typescript
afterBundling(inputDir: string, outputDir: string): string[] {
    return [
        'npx prisma generate', 
        'rm -rf node_modules/@prisma/engines', 
        'rm -rf node_modules/@prisma/client/node_modules', 
        'rm -rf node_modules/.bin', 
        'rm -rf node_modules/prisma'
    ]
}
```

I should probably add `prisma-appsync` to the list as well!

**Comment by maoosi** [comment]
Resolved with [1.0.0-beta.52](https://github.com/maoosi/prisma-appsync/releases/tag/1.0.0-beta.52)
